dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sme/SmeData.kt

```diff
@@ -0,0 +1,86 @@
+package org.dataland.datalandbackend.model.sme
+
+import org.dataland.datalandbackend.annotations.DataType
```

Comment:
Bug in first entry of SfdrData "fiscalYear" should not be string but may only be "Deviation" or "No Deviation"

Reply:
In EuTaxonomy ist es auch falsch, fix ist in Arbeit